src/uu/date/src/date.rs

```diff
@@ -294,6 +303,23 @@ pub fn uumain(args: impl uucore::Args) -> i32 {
     0
 }
 
+#[cfg(any(windows, all(unix, not(target_os = "macos"))))]
+/// Apply platform specific matcher arguments to the app given.
+fn make_platform_app<'a, 'b>(app: App<'a, 'b>) -> App<'a, 'b> {
```

Comment:
I wonder if this isn't overkill ?
I think the option should also exist on mac but in the help section, add "not available on mac yet"
and this function should be added back in the main app declaration